components/automate-cli/cmd/chef-automate/status.go

```diff
@@ -75,6 +119,41 @@ func getStatus() (*api.StatusResponse, error) {
 	return res, nil
 }
 
+func runStatusSummaryCmdFunc(statusSummaryCmdFlags *StatusSummaryCmdFlags) func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		return executeStatusSummary(cmd, args, statusSummaryCmdFlags)
+	}
+}
+
+func executeStatusSummary(cmd *cobra.Command, args []string, statusSummaryCmdFlags *StatusSummaryCmdFlags) error {
+	if isA2HARBFileExist() {
+		infra, err := getAutomateHAInfraDetails()
+		if err != nil {
+			return err
+		}
+		statusSummary := NewStatusSummary(infra, FeStatus{}, BeStatus{}, 10, time.Second, statusSummaryCmdFlags)
+		sshUtil := statusSummary.(*Summary).setSSHConfig()
+
+		err = statusSummary.Run(sshUtil)
```

Comment:
calling the `Run` function here creates confusion, as below we are showing status with if conditions, this function name should be `prepare` or something like this